client-app/src/common/models/error-info.model.ts

```diff
@@ -0,0 +1,17 @@
+import { Observable } from 'rxjs';
```

Comment:
We use types.ts for all domain definitions in other branches, maybe you will follow by this rule?

Reply:
Added types.ts